skbio/diversity/alpha/_base.py

```diff
@@ -311,6 +294,59 @@ def esty_ci(counts):
     return f1 / n - z * np.sqrt(W), f1 / n + z * np.sqrt(W)
 
 
+@experimental(as_of="0.4.0-dev")
+def faith_pd(counts, otu_ids, tree, suppress_validation=False):
```

Comment:
Comment from @ebolyen: change ``suppress_validation=False`` to ``validate=True`` through-out, for consistency with the rest of the codebase.